app/backend/src/clj/teet/meeting/meeting_commands.clj

```diff
@@ -204,15 +205,52 @@
          "/meetings/" activity-eid
          "/" meeting-eid)))
 
+
+(defn historical-meeting-notify [db meeting from-user participants project-eid]
+  (log/debug "from-user is" from-user)
+  (tx-ret
+   (vec
+    (for [to-user participants]
+      (notification-db/notification-tx
+       db
+       (do
+         (log/debug "to-user is" to-user)
+           
+         {:from from-user
+          :to to-user
```

Comment:
what if there are participants who are not registered in teet, will to list have those?

Reply:
yes, but it works out since those users also seem to have user objects in db